### ISSUE TITLE
Add PNG/GeoTiff RGB composite images to docs 

### ISSUE THREAD

**Issue by vpipkt** [locationtech/rasterframes#217]
In some places in the docs, such as ML chapters in particular, it will be nice to display an RGB composite of  (a portion of) the study area. In ML chapters makes a nice comparison to the predicted land cover.

Look for other places where the larger area preview afforded by the gtiff writer would be nice to have.

Depends on #145

**Comment by vpipkt** [comment]
Related/dupe of #258  #289 #351
